x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-xml-maven.adoc

```diff
@@ -0,0 +1,81 @@
+= X12 EDI - XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio.
+
+If you manually code a Mule runtime engine (Mule) app in XML either from the Anypoint Studio XML editor or from a text editor, you can access the connector from your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
+
+When you add the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location and it also adds a dependency to the `pom.xml` file.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `mule` tag of the header
+of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/x12-edi
+http://www.mulesoft.org/schema/mule/x12-edi/current/mule-x12-edi.xsd
+----
+
+This example shows how the namespace statements are placed in the XML:
+
+[source,xml,linenums]
+----
+<mule xmlns:$2="http://www.mulesoft.org/schema/mule/$2"
+	xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:file="http://www.mulesoft.org/schema/mule/file"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/file
+	http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
+	http://www.mulesoft.org/schema/mule/http
+	http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/ee/core
+	http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+	http://www.mulesoft.org/schema/mule/$2
+	http://www.mulesoft.org/schema/mule/$2/current/mule-$2.xsd">
+----
+
+
+== Add a POM File Dependency
+
+The Apache Maven `pom.xml` file generated by Anypoint Studio contains dependencies
+for a Mule app. When you code a Mule app manually, include this XML snippet in
+your `pom.xml` file to enable access to this connector:
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-x12-connector</artifactId>
+  <version>2.5.3</version>
```

Comment:
You can just leave version as x.x.x so you don't have to update it with every release.